### PR TITLE
MNTR-18 progress bar shared component

### PR DESCRIPTION
**Pull request by Gokselagan**

### CODE REVIEW

File: src/shared-components/Visual/ProgressBar/ProgressBar.tsx
Comment (annasvst) [inline src/shared-components/Visual/ProgressBar/ProgressBar.tsx:1]: Rename the file to `index.tsx` to simplify imports

File: src/shared-components/Visual/ProgressBar/ProgressBar.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { theme } from 'src/styles/theme';
+
+interface ProgressBarProps {
+  trackColor: string;
+  progressValue: number;
+  railColor: string;
+  height: number;
+}
+
+export const ProgressBar = ({
```
Comment (annasvst) [inline src/shared-components/Visual/ProgressBar/ProgressBar.tsx:11]: Only ask for the `color` and `progressValue`. Set up other colors directly in the component. No need to be able to customize the height as we want a shared component that has consistent look when used across the app.

File: src/shared-components/Visual/ProgressBar/ProgressBar.tsx
Comment (Gokselagan) [reply src/shared-components/Visual/ProgressBar/ProgressBar.tsx:1]: Done.

File: src/shared-components/Visual/ProgressBar/ProgressBar.tsx (hunk 1 above)
Comment (Gokselagan) [reply src/shared-components/Visual/ProgressBar/ProgressBar.tsx:11]: Fixed

**Review by annasvst — DISMISSED** [review]
Approved. Merge the MNTR-17 pull request before this one. You'll probably have a merge conflict dues to the Slider component. Make sure that you accept whatever was added in the other branch here.

**Review by annasvst — APPROVED** [review]

**Review by annasvst — APPROVED** [review]

**Pull request merged**